services/storage/src/simcore_service_storage/datcore_adapter/datcore_adapter.py

```diff
@@ -64,7 +64,15 @@ async def _request(
             params=params,
             **request_kwargs,
         ) as response:
-            return await response.json()
+            response_data = await response.json()
+            assert isinstance(
+                response_data,
+                (
+                    dict,
+                    list,
+                ),
+            )  # nosec
```

Comment:
can we get away with a one liner here?
```suggestion
            assert isinstance(response_data, (dict, list))  # nosec
```